src/PhpSpreadsheet/Chart/DataSeriesValues.php

```diff
@@ -60,9 +60,9 @@ class DataSeriesValues
     private $dataValues = [];
 
     /**
-     * Fill color.
+     * Fill color (can be array with colors if dataseries have custom colors).
      *
-     * @var string
+     * @var array|string
```

Comment:
This should be and all similar type hinting too:

```suggestion
     * @var string[]|string
```